### PR TITLE
Update free-courses-hi.md for DevOps

### PR DESCRIPTION
**Pull request by sonali-rajput**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resources for DevOps.

### Description
A tutorial on DevOps for Beginners.

### Why is this valuable (or not)?
Complete tutorial on DevOps for Beginners in Hindi.
### How do we know it's really free?
It is a YouTube tutorial.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a YouTube tutorial.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -184,6 +184,7 @@
 ### DevOps
 
 * [DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
+* [Devops Tutorial For Beginners In Hindi | DevOps Tutorial | DevOps Full Course](https://youtu.be/tgmM3_2dZwg) - Great Learning
```
Comment (eshellman) [inline courses/free-courses-hi.md:187]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes 
No shortened links

**Comment by sonali-rajput** [comment]
Even after swapping as the error check says, It again ask me to swap the same line , I've changed it so many times..

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -184,15 +184,16 @@
...
 * [DevOps Tutorials in Hindi/Urdu \| Devops सीखने का सबसे आसान तरीका \| Complete Devops including Git,Jenkins,Maven,Chef,docker,Ansible and Kubernetes](https://www.youtube.com/playlist?list=PLBGx66SQNZ8aPsFDwb79JrS2KQBTIZo10) - Bhupinder Rajput, Technical Guftgu
 
 
 ### Flutter
 
 * [Complete Flutter Tutorial In Hindi By Desi Programmer](https://youtube.com/playlist?list=PLlFwzkUNmr94BF0KH7BYPL7DsZjhJRdTm) - Desi Programmer
 * [Flutter 3 tutorial in Hindi](https://youtube.com/playlist?list=PLB97yPrFwo5g-XcPlfSXSOeeby23jVAcp) - CODERS NEVER QUIT
-* [Flutter Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfGLShoW0vVX_tcyT8u1Y3E) - WsCube Tech
 * [Flutter Essential Course](https://www.youtube.com/watch?v=Wdx8vwN_nWI) - freecodecamp (YouTube)
+* [Flutter Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfGLShoW0vVX_tcyT8u1Y3E) - WsCube Tech
```
Comment (bugbunnyy) [inline courses/free-courses-hi.md:196]: This was previously located in the correct position.

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -184,15 +184,16 @@
 ### DevOps
 
 * [DevOps Training Videos in Hindi](https://www.youtube.com/playlist?list=PLQbQOmlGYH3sxlq9ugoq1ipNFP7tus5Gd) - edureka! Hindi
+* [DevOps Training Tutorials For Beginners in Hindi](https://youtube.com/playlist?list=PLYEK_dHOjwtODYB46wFuc34muw9Gl5X5x) - Linux Wale Guruji
```
Comment (bugbunnyy) [inline courses/free-courses-hi.md:187]: Swap this and above line.

**Comment by bugbunnyy** [comment]
Hope this will solve your linter warnings.

**Pull request closed without merging**